tests/cephfs/cephfs_utilsV1.py

```diff
@@ -2972,6 +2980,16 @@ def wait_for_host_online(self, client1, node):
                 if hostname_status == "Offline":
                     raise CommandFailed("Host is in Offline state")
 
+    @retry(CommandFailed, tries=5, delay=30)
+    def wait_for_host_offline(self, client1, node):
```

Comment:
Do we have another method for waiting for host to be online? If not, can we enhance the same method to check for wait_for_host by passing online/offline status to reuse the function for 2 scenarios?

Reply:
Yes neha we have both methods (wait_for_host_online, Wait_for_host_offline)
To make it easy to reuse i have just came up with 2 methods so that user will not have confusion of what parameter to pass he can simply use the function name